packages/carbon-graphs/tests/unit/controls/Gantt/GanttEventline-spec.js

```diff
@@ -24,6 +24,8 @@ import {
 describe('Eventline', () => {
   let ganttChartContainer;
   let axisObj;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Do we need this line?

Reply:
agreed?  I can't think of a case where we would need/want to keep unused variables in the code

Reply:
Actually, i added **gantt** variable to assign to all new objects as linter suggested **(gantt = new Gantt(axisObj);)**. After that linter came up with a new error as shown in the below image. Since we do not have any use of **gannt** variable (except to assign it to the new objects) i have to suppress the linter error as needed.  
![image](https://user-images.githubusercontent.com/63670637/104773520-ca60e700-573a-11eb-8b76-df543ac8dd38.png)